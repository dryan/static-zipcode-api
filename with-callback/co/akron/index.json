{"region": {"fips": "08", "name": "Colorado", "abbr": "CO"}, "postal_codes": ["80720"], "name": "Akron", "counties": [{"fips": "121", "name": "Washington"}]}