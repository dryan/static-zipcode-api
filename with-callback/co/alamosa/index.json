{"region": {"fips": "08", "name": "Colorado", "abbr": "CO"}, "postal_codes": ["81102", "81101"], "name": "Alamosa", "counties": [{"fips": "003", "name": "Alamosa"}, {"fips": "021", "name": "Conejos"}]}