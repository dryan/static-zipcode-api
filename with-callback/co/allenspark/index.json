{"region": {"fips": "08", "name": "Colorado", "abbr": "CO"}, "postal_codes": ["80510"], "name": "Allenspark", "counties": [{"fips": "013", "name": "Boulder"}]}