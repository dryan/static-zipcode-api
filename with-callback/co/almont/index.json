{"region": {"fips": "08", "name": "Colorado", "abbr": "CO"}, "postal_codes": ["81210"], "name": "Almont", "counties": [{"fips": "051", "name": "Gunnison"}]}