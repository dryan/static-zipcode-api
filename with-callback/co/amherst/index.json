{"region": {"fips": "08", "name": "Colorado", "abbr": "CO"}, "postal_codes": ["80721"], "name": "Amherst", "counties": [{"fips": "095", "name": "Phillips"}]}