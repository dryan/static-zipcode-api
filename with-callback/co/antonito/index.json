{"region": {"fips": "08", "name": "Colorado", "abbr": "CO"}, "postal_codes": ["81120"], "name": "Antonito", "counties": [{"fips": "021", "name": "Conejos"}]}