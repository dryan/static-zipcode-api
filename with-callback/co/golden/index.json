{"region": {"fips": "08", "name": "Colorado", "abbr": "CO"}, "postal_codes": ["80401", "80439", "80403", "80402", "80419"], "name": "Golden", "counties": [{"fips": "047", "name": "Gilpin"}, {"fips": "005", "name": "Arapahoe"}, {"fips": "019", "name": "Clear Creek"}, {"fips": "059", "name": "Jefferson"}, {"fips": "013", "name": "Boulder"}]}